src/App.tsx

```diff
@@ -1,12 +1,52 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import cn from 'classnames';
+import { Footer } from './components/Footer';
+import { Filter } from './types/Filter';
+import { getFilteredList } from './utils/getFilteredList';
+import { errorMessages, ErrorMessages } from './types/ErrorMessages';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todosList, setTodosList] = useState<Todo[] | null>(null);
```

Comment:
```suggestion
  const [todosList, setTodosList] = useState<Todo[]>([]);
```
You can set an empty array by default to simplify your logic